source/github-events/on-new-review.ts

```diff
@@ -0,0 +1,13 @@
+import delegate from 'delegate-it';
+
+function callbackWrapper(callback: VoidFunction): (event: CustomEvent) => void {
```

Comment:
If you don't memoize this call, then `delegate` will add a new listener every time `onNewReviewComment` is called. Event listeners are natively deduplicated if the same exact function is passed to it. Something like:
```suggestion
const callbackWrapper = mem((callback: VoidFunction): (event: CustomEvent) => void => {
```